projects/batfish-common-protocol/src/org/batfish/datamodel/Interface.java

```diff
@@ -173,17 +187,22 @@ else if (name.startsWith("TenGigabitEthernet")) {
          return InterfaceType.PHYSICAL;
       }
       else if (name.startsWith("Tunnel")) {
-         return InterfaceType.VPN;
```

Comment:
It is actually the other way: VPN (encrypted) is more restricted than Tunnel (encapsulation, that covers VPNs along with many non-VPN cases). See IOS 12.4 documentation: http://www.cisco.com/c/en/us/td/docs/ios/12_4/interface/configuration/guide/inb_tun.html and a Tunnel interface configured for IPSec VPN at: http://www.cisco.com/en/US/technologies/tk583/tk372/technologies_white_paper0900aecd8029d629_ps6635_Products_White_Paper.html

The code does not seem to have any dependencies on InterfaceType.VPN for Tunnel interfaces for Cisco configurations; there is one reference for Juniper, that does not apply in this case.

If the current code has de